docs/standard/commandline/snippets/get-started-tutorial/csharp/Stage2/scl.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.CommandLine" Version="2.0.0-beta2.21617.1" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="sampleQuotes.txt">
```

Comment:
It might be simpler to represent this as a `<Content Include="sampleQuotes.txt">`? Content items are included in the output directory by default.

Reply:
When I change to content include and build the app, sampleQuotes.txt doesn't show up in bin/debug/net6.0.

Reply:
It works with content include if I keep the CopyAlways line.